src/assets/scss/_faceting.scss

```diff
@@ -36,10 +36,23 @@
         padding: 10px 5px;
         text-transform: none;
         position: relative;
+        // Overriding accordian-button css to align right angle bracket icon to left
+        flex-direction: row-reverse;
+        justify-content: flex-end;
 
         background-color: $table-header-background-color;
         color: $black-color;
 
+        // Overriding accordian-button css to align right angle bracket icon to left
+        &::after {
+          margin: 0px 8px;
```

Comment:
Given that the size and color of the icon is not the same as master, instead of just transforming to fix the direction, you should just show the icon that we want instead of the svg that bootstrap uses.

Something like this
```scss
&::after {
    background-image: none;
    content: '\f078'; // fa-solid fa-chevron-down
    font-family: "Font Awesome 6 Free";
    font-weight: 900;
    // TODO change the font-size and position to make sure it looks the same as master
}

&:not(.collapsed)::after {
  content: '\f054'; // fa-solid fa-chevron-right
  // TODO change the font-size and position to make sure it looks the same as master
}
```